src/components/layout/RightSection.vue

```diff
@@ -55,10 +56,26 @@
       FiatValue
     },
     data() {
-      return {}
+      return { 
+        browser: detect()
+      }
     },
     computed: {
       ...mapGetters(['topics', 'isLoggedIn']),
+      downloadUrl () {
+        switch(this.browser && this.browser.name) {
```

Comment:
this shouldn't work, switch on a boolean?

Reply:
It works actually, but if you are not happy with this - I can change the implementation.

Reply:
just wrap it in an if(this.browser) :)